SpiffyTitles/plugin.py

```diff
@@ -129,9 +127,14 @@ def handler_dailymotion(self, url, info, channel):
 
                     if response is not None and "title" in response:
                         video = response
-                        dailymotion_template = \
-                            Template(self.registryValue("dailymotionVideoTitleTemplate",
-                                                        channel=channel).decode("utf-8"))
+                        if sys.version_info[0] >= 3:
+                            dailymotion_template = Template(
+                                self.registryValue("dailymotionVideoTitleTemplate",
+                                                   channel=channel))
+                        else:
+                            dailymotion_template = Template(
+                                self.registryValue("dailymotionVideoTitleTemplate",
+                                                   channel=channel).decode("utf-8"))
```

Comment:
Maybe we could write a method for this. This bit of code is repeated for each template type, so I think it's worth refactoring a bit. Consider the following bit of psuedocode:

`
def get_template (self, **kwargs):
  is_py3 = sys.version_info[3] >= 3
  # return template based on channel, name, and whether we're using py3 here
  # and conditionally apply .decode("utf-8")
`